styles.css

```diff
@@ -0,0 +1,137 @@
+* {
+    Box-sizing: Border-box
+}
+
+html {
+    background-image: url('img/backgroundImage.jpg');
+    background-color: #000000;
+    background-size: 1920px 1080px;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+
+
+    -moz-osx-font-smoothing: grayscale;
```

Comment:
This line of code does not look like to make any difference to the text in the texts.

Reply:
*the way the text looks